### ISSUE TITLE
Support for `import "my/file.js;`

### ISSUE THREAD

**Issue by alber70g** [guybedford/es-module-shims#11]
I'm facing an issue where bare imports aren't resolved using the import map. I'm not sure if this is behaviour that is compliant with the spec, so please enlighten me if that is the case.

In my components we use bare imports for web-components that register themselves as `customElements.define(...)`. 

So we import them as such: `import "ui-lib/ui-button.js"`. 
In the import-map we provide them bundled as: `"ui-lib/ui-button.js": "ui-lib/bundle.js"`. However it seems that these aren't parsed and resolved by the shim.

**Comment by alber70g** [comment]
So, this was quite a challenge, but I think I have found a way to resolve this:

`lexer.js:157` should add:

```js
          && str.charCodeAt(i + 6) !== 34/*"*/ 
          && str.charCodeAt(i + 6) !== 36/*'*/
```
Otherwise, it somehow skips the import and doesn't parse `import"some/import.js";`

**Comment by guybedford** [comment]
Oh wow, great catch, thank you so much for digging that up. If you've found the line, any chance of posting a test into https://github.com/guybedford/es-module-shims/blob/master/test/unit.js with the fix? Otherwise I will get to this when I can.

**Comment by guybedford** [comment]
I've added this fix and released in 0.2.7.

Thanks again for the report, sorry it took some digging.
